frontend/src/pages/Auth/SignUp.tsx

```diff
@@ -78,36 +79,39 @@ function SignUp() {
   };
 
   return (
-    <StyledSignupForm onSubmit={handleRegister}>
-      {!githubUser && (
-        <>
-          <CustomInput
-            name="nickname"
-            type="text"
-            placeholder="영문, 숫자 조합 10자 이하"
-          />
-          <CustomInput
-            name="password"
-            type="password"
-            placeholder="영문/특문/숫자 조합 16자 이하"
-          />
-        </>
-      )}
-      <CustomInput
-        type="text"
-        placeholder="시∙구 제외, 동만 입력."
-        value={selectedRegion.name}
-        readOnly={true}
-      />
-      <ul>
-        {region.map((item) => (
-          <li key={item.id} onClick={() => handleSelectRegion(item)}>
-            {item.name}
-          </li>
-        ))}
-      </ul>
-      <button type="submit">회원가입</button>
-    </StyledSignupForm>
+    <>
+      <PageHeader pageName="회원가입" />
+      <StyledSignupForm onSubmit={handleRegister}>
+        {!githubUser && (
+          <>
+            <CustomInput
+              name="nickname"
+              type="text"
+              placeholder="영문, 숫자 조합 10자 이하"
+            />
+            <CustomInput
+              name="password"
+              type="password"
+              placeholder="영문/특문/숫자 조합 16자 이하"
+            />
+          </>
+        )}
+        <CustomInput
+          type="text"
+          placeholder="시∙구 제외, 동만 입력."
+          value={selectedRegion.name}
+          readOnly={true}
```

Comment:
readOnly를 추후에 동적으로 핸들링하시려고 이렇게 두신걸까욥!?
아니라면 다음과 같이 표현해도 좋겠습니다~!!
```suggestion
          readOnly
```

Reply:
readOnly가 상황따라 다르게 될 것 같았는데 지금보니 고정으로 해도 될 것 같네요 자동완성할 때 필요하니깐 그때 같이 수정하겠습니다!